src/EnergyPlus/HVACUnitarySystem.cc

```diff
@@ -10987,13 +11094,21 @@ namespace HVACUnitarySystem {
 		InletNode = UnitarySystem( UnitarySysNum ).UnitarySystemInletNodeNum;
 
 		if ( SpeedNum > 1 ) {
-			AverageUnitMassFlow = CompOnMassFlow;
+			if( CoolingLoad && MultiOrVarSpeedCoolCoil( UnitarySysNum ) || HeatingLoad && MultiOrVarSpeedHeatCoil( UnitarySysNum ) ) {
```

Comment:
@rraustad The clang tests are throwing four warnings like this:
'&&' within '||' [-Wlogical-op-parentheses]
This line looks like it needs some clarification with parenetheses.  Also 11107 below.  Not sure why four warnings.